colossus-tests/src/test/scala/colossus/protocols/http/client/StreamingHttpResponseParserSpec.scala

```diff
@@ -109,21 +138,23 @@ class StreamingHttpResponseParserSpec extends WordSpec with MustMatchers with Tr
       }
 
     }
-    "decode a response that was encoded by colossus with no body" in {
+    "decode a response that was encoded by colossus with no body" ignore {
       validateEncodedStreamingHttpResponse(ByteString(""))
     }
 
-    "decode a response that was encoded by colossus with a body" in {
```

Comment:
We have several ignored tests in this spec..if they are no good, we should just delete them.  They look like they would be useful however.